src/python/install.sh

```diff
@@ -306,66 +347,19 @@ add_symlink() {
     fi
 }
 
-install_openssl11() {
+install_openssl3() {
     local _prefix=$1
-    mkdir /tmp/openssl11
+    mkdir /tmp/openssl3
     (
-    cd /tmp/openssl11
-    curl -L -f -O https://www.openssl.org/source/openssl-1.1.1w.tar.gz
-    tar xzf openssl-1.1.1w.tar.gz
-    cd openssl-1.1.*
-    ./config --prefix=${_prefix} --openssldir=${_prefix}
-    make -j $(nproc)
-    make install_dev
+        cd /tmp/openssl3
+        curl -L -f -O https://www.openssl.org/source/openssl-3.0.8.tar.gz
```

Comment:
I am worried that installing a specific version of OpenSSL would quickly make this stale and this Feature won't benefit from security patches which gets released. 

Any thoughts about using 👇  and finding the latest version of OpenSSL (for major version 3)? 

https://github.com/devcontainers/features/blob/baf1568d80bb6f1eb73dd7e1b774b2fa5f3b662d/src/python/install.sh#L218